megalinter/descriptors/sql.megalinter-descriptor.yml

```diff
@@ -26,19 +26,20 @@ linters:
   - linter_name: sqlfluff
     linter_url: https://www.sqlfluff.com/
     linter_repo: https://github.com/sqlfluff/sqlfluff
-    linter_banner_image_url: https://raw.githubusercontent.com/sqlfluff/sqlfluff/master/images/sqlfluff-wide.png
+    linter_banner_image_url: https://raw.githubusercontent.com/sqlfluff/sqlfluff/main/images/sqlfluff-wide.png
     linter_image_url: https://docs.sqlfluff.com/en/stable/_static/images/sqlfluff-lrg.png
     linter_rules_url: https://docs.sqlfluff.com/en/stable/rules.html
     linter_rules_configuration_url: https://docs.sqlfluff.com/en/stable/configuration.html
+    config_file_name: .sqlfluff
     config_file_name: setup.cfg
```

Comment:
@tunetheweb only one config_file_name, you may want to remove the one with setup.cfg :)

Reply:
```suggestion
```